test-app/tsconfig.json

```diff
@@ -28,7 +28,10 @@
       {
         "transform": "functionless/lib/compile"
       }
-    ]
+    ],
+    "paths": {
+      "functionless": ["../"]
```

Comment:
Hmm what does this do? I've been using yarn link, does this improve that experience?

Reply:
Oh there might be something wrong with my environment then because I kept having typescript errors here until I added this. What yarn version should I use with this project?

Reply:
I removed it with latest commit, I think it was something specific to my setup